tools/fast/environment/environment_memory_genesis.go

```diff
@@ -220,6 +221,7 @@ func (e *MemoryGenesis) startGenesisServer() error {
 	}
 
 	e.genesisServerAddr = ln.Addr().String()
+	e.Log().Infof("Genesus URL: http://%s%s", e.genesisServerAddr, handlePath)
```

Comment:
This is a one off change unrelated to the rest of the PR, added via separate commit.